Code/Framework/AzCore/AzCore/DOM/DomPath.h

```diff
@@ -68,6 +68,8 @@ namespace AZ::Dom
     class Path final
     {
     public:
+        AZ_TYPE_INFO(Path, "{C0081C45-F15D-4F46-9680-19535D33C312}")
```

Comment:
Could you add the namespace of `AZ::Dom::Path` here?
We actually have TypeInfo for [AZ::IO::Path](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/IO/Path/Path.h#L740-L744) and using the fully qualified namespace would help disambiguate

Reply:
sure, will do